### ISSUE TITLE
Time Options

### ISSUE THREAD

**Issue by filipeangelo** [krijnsent/crypto_vba#27]
I was wondering if its possible to get a option  to aggregate time ex: 5m,15m,2h,7d  
Thank you!!

**Comment by krijnsent** [comment]
Hi, what time options are you looking for? The functions I created for the cryptocompare API?

**Comment by filipeangelo** [comment]
Hi, I actually could use minutes. I've added a code line in the vba. and
also added aggregation formulas in the sheets. if anyone is interested I
can send ( I am not a programmer so I dont think its good to open a branch
of it is pretty my much excel formulas)

2018-02-11 10:15 GMT-02:00 Koen Rijnsent <notifications@github.com>:

**Comment by krijnsent** [comment]
Hi, could you please mention the module you are talking about and add the line(s) you created? Please add them to this issue, that would help big time.

**Comment by filipeangelo** [comment]
Hi I've changed the function :
C_ARR_OHLCV in the following code:


If UCase(DayHour) = "D" Then
    cmd = "histoday"
ElseIf UCase(DayHour) = "H" Then
    cmd = "histohour"
'New Lines
ElseIf UCase(DayHour) = "M" Then
    cmd = "histominute"
'End of new lines

With that I was able to download in minutes. with some excel formulas I can
aggregate time to 5m,10m 2h, 4h 7days etc. But its nothing automatic.

2018-02-13 5:15 GMT-02:00 Koen Rijnsent <notifications@github.com>:
